src/main/java/Events.java

```diff
@@ -0,0 +1,13 @@
+public class Events extends Task {
+    private String time;
+
+    public Events(String description, String time) {
+        super(description);
+        this.time = time;
+    }
+
+    @Override
+    public String getTime() {
+        return String.format(" (at: %s)", time);
```

Comment:
can consider overriding your toString method to return your output message instead